### ISSUE TITLE
Element objects are not ellipsizing

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmctweb#498]
Observed in VISTA.

**Comment by akhenry** [comment]
What's the expected behavior here? I don't know what ellipsize means in the case of a web page component

**Comment by charlesh88** [comment]
Sorry, this won't mean much to anyone but me. The problem is CSS related.

**Comment by shobart** [comment]
I *think* this has been corrected, but it's hard to tell given the description.  Here's what I did:

1. Started editing "AmandaLayout" (i.e. navigated to My Items -> AMCook -> AmandaLayout, and selected the pencil icon)
2. Played around with re-sizing the window and the Elements Area (lower part of right-hand side).  Observed that "NIRVSS Bracket & PCB Temps" gets an ellipsis (...) added to the end when the Elements area is shrunk far enough to truncate it.

If that's the issue, then this appears to be resolved.
